### DISCUSSION TITLE
Export builtin templates

### DISCUSSION THREAD

**Discussion by baahrens** [stevearc/overseer.nvim#235] [Q&A]
Hi Steven,

brilliant plugin, thank you so much!

Just wanted to ask: Would you be open to a PR that adds a `overseer.builtin` (or similar) export to the plugin, so that one can adjust the builtin templates before using them in `setup`? That would make it possible to e.g. adjust the name of the builtin templates or enforce a specific package manager for `npm` scripts.

**Comment by stevearc** [comment]
If you need to make adjustments to the built-in templates, you could do this a couple ways:
1. `require` them yourself and monkey patch in whatever changes you want
2. copy them to your own config under a new path and make your changes there. Then pass in an explicit list to `templates`, replacing the built-in with the path to your module.
